it/conduct/index.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: page
+title: "Il codice di condotta della comunità di Ruby"
+lang: it
+---
+Il presente codice di condotta è stato scelto basandosi su una proposta di bozza per il codice di condotta di PostgreSQL, per mantenere la collaborazione nella comunità degli sviluppatori Ruby sicura e produttiva.
+Ogni cosa collegata alla comunità di Ruby (conferenze ecc.) dovrebbe avere il suo codice di condotta.
+{: .summary}
+
+Questo documento fornisce delle linee guida di comunità per ottenere un luogo sicuro, dove c'è rispetto, produttivo e collaborativo per ogni persona che voglia contribuire alla comunità di Ruby. Si applica a tutti gli spazi collaborativi, come i canali di comunicazione comunitaria (mailing lists, patch inviate, commenti ai commit ecc.).
+
+ * I partecipanti devono essere tolleranti nei confronti delle opinioni contrastanti.
+ * I partecipanti devono assicurare che il loro linguaggio e le loro azioni siano esenti da attacchi personali e osservazioni personali denigratorie.
+ * Quando si interpretano le parole o le intenzioni degli altri, i partecipanti devono presumere la buona fede.
+ * Comportamenti che possono essere identificati come molestie non saranno tollerati.
```

Comment:
This sentence is missing `ragionevolmente`, right?

Reply:
@kennyadsl Sì